### PR TITLE
chore(aws-sdk): implement SQS/SNS precise spans

### PR DESCRIPTION
**Pull request by YanivD**

### CODE REVIEW

**Comment by YanivD** [comment]
**left todo:**
* cover with tests
* verify no more than 10 meta attributes

File: instrumentation/aws_sdk/lib/opentelemetry/instrumentation/aws_sdk/handler.rb [hunk 1]
```diff
@@ -48,7 +67,33 @@ def get_operation(context)
...
+          end
+        end
+
         def get_span_name(context)
+          client_method = get_client_method(context)
+          case client_method
+          when SQS_SEND_MESSAGE, SQS_SEND_MESSAGE_BATCH
+            "#{MessagingHelper.get_queue_name(context)} send"
+          when SNS_PUBLISH
+            "#{MessagingHelper.get_queue_name(context)} Publish"
```
Comment (blumamir) [inline instrumentation/aws_sdk/lib/opentelemetry/instrumentation/aws_sdk/handler.rb:88]: according to the spec this should be "send" as well

Comment (YanivD) [reply instrumentation/aws_sdk/lib/opentelemetry/instrumentation/aws_sdk/handler.rb:88]: ok

Comment (YanivD) [reply instrumentation/aws_sdk/lib/opentelemetry/instrumentation/aws_sdk/handler.rb:88]: @blumamir 

https://github.com/open-telemetry/opentelemetry-js-contrib/blob/main/plugins/node/opentelemetry-instrumentation-aws-sdk/src/services/sns.ts#L46-L48

**Pull request closed without merging**